### PR TITLE
automatically fixed all private-{bin,etc} lines

### PR DESCRIPTION
**Pull request by rusty-snake**
Bring all private-etc and private-bin lines into an alphabetical order, using this:
https://gist.github.com/rusty-snake/a1010a3daf3c54e93dfe03f2f5ce3d96

* special characters are ignored
* the script order case insensitive

Missing (will not be implemented, only for completeness): 
 * commented lines
   * `#private-{bin,etc}`, `# private-{bin,etc}`
   * What about something like `# private-etc foo,bar -- breaks baz` and
`# private-etc must first be enabled in foobar`
 * `private-lib`
 * see also https://github.com/netblue30/firejail/issues/2739#issuecomment-499892002
 * review

### CODE REVIEW

File: etc/kid3.profile [hunk 1]
```diff
@@ -37,7 +37,7 @@ tracelog
 
 private-cache
 private-dev
-private-etc alternatives,drirc,fonts,kde5rc,gtk-3.0,dconf,machine-id,ca-certificates,ssl,pki,hostname,hosts,resolv.conf,pulse,,crypto-policies
+private-etc ,alternatives,ca-certificates,crypto-policies,dconf,drirc,fonts,gtk-3.0,hostname,hosts,kde5rc,machine-id,pki,pulse,resolv.conf,ssl
```
Comment (rusty-snake) [inline etc/kid3.profile]: Note for me: remove the trailing comma

**Comment by Vincent43** [comment]
I wonder hot to automate this and make mandatory as it's very likely that future profile updates wil break the order.

**Comment by rusty-snake** [comment]
@Vincent43 I'm not sure if I understood you correctly, but that's fix every profile for now, and if in the future something like that will be enforced I can't say.

**Comment by Vincent43** [comment]
Yes, this will fix current state but I doubt people would care about alphabetic order in profiles while creating future PRs so this is quite cat and mouse game. Nevertheless I have nothing against this PR.

**Comment by glitsj16** [comment]
@rusty-snake Thanks for mentioning your [sort.py](https://gist.github.com/rusty-snake/a1010a3daf3c54e93dfe03f2f5ce3d96) script. I extended it so it handles private-lib too. Why not offer it up for `contrib`? We could use more of these helper scripts IMHO. Great job :+1: .

**Comment by Vincent43** [comment]
I think this script may better fit for travis CI, i.e. it could fail if there is diff between its output and proposed patch in PR.

**Comment by rusty-snake** [comment]
@glitsj16 I also add support for `private-lib` and improve the performance.

**Comment by rusty-snake** [comment]
For @Vincent43 proposal lets move back to #2739.

**Pull request merged**